Changed some file names

dlraffleck opened the pull request:
Just changed a few file names/directories in the first R script.  Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

Commit:
Changed some file names

gcgaines commented:
Changes changed back.

Also, I made a BitBucket account, username = gcgaines

---

From: dlraffleck notifications@github.com
Sent: Tuesday, February 09, 2016 2:01 PM
To: gcgaines/laser_lef
Subject: [laser_lef] Changed some file names (#1)

Just changed a few file names/directories in the first R script. Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

---

You can view, comment on, or merge this pull request online at:

  https://github.com/gcgaines/laser_lef/pull/1

[https://avatars2.githubusercontent.com/u/16769852?v=3&s=400]https://github.com/gcgaines/laser_lef/pull/1

Changed some file names by dlraffleck · Pull Request #1 · gcgaines/laser_lefhttps://github.com/gcgaines/laser_lef/pull/1
github.com
Just changed a few file names/directories in the first R script. Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

Commit Summary
-   Changed some file names

File Changes
-   M 1.ReadRIPdb.2015dec14.Rhttps://github.com/gcgaines/laser_lef/pull/1/files#diff-0 (6)

Patch Links:
-   https://github.com/gcgaines/laser_lef/pull/1.patch
-   https://github.com/gcgaines/laser_lef/pull/1.diff

## 

Reply to this email directly or view it on GitHubhttps://github.com/gcgaines/laser_lef/pull/1.

dlraffleck commented:
Check your bitbucket account and you should see a project called IdahoPSPs…

From: gcgaines [mailto:notifications@github.com]
Sent: Wednesday, February 10, 2016 5:58 PM
To: gcgaines/laser_lef
Cc: Affleck, David
Subject: Re: [laser_lef] Changed some file names (#1)

Changes changed back.

Also, I made a BitBucket account, username = gcgaines

---

From: dlraffleck notifications@github.com
Sent: Tuesday, February 09, 2016 2:01 PM
To: gcgaines/laser_lef
Subject: [laser_lef] Changed some file names (#1)

Just changed a few file names/directories in the first R script. Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

---

You can view, comment on, or merge this pull request online at:

https://github.com/gcgaines/laser_lef/pull/1

[https://avatars2.githubusercontent.com/u/16769852?v=3&s=400]https://github.com/gcgaines/laser_lef/pull/1

Changed some file names by dlraffleck · Pull Request #1 · gcgaines/laser_lefhttps://github.com/gcgaines/laser_lef/pull/1
github.com
Just changed a few file names/directories in the first R script. Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

Commit Summary
- Changed some file names

File Changes
- M 1.ReadRIPdb.2015dec14.Rhttps://github.com/gcgaines/laser_lef/pull/1/files#diff-0 (6)

Patch Links:
- https://github.com/gcgaines/laser_lef/pull/1.patch
- https://github.com/gcgaines/laser_lef/pull/1.diff

## 

Reply to this email directly or view it on GitHubhttps://github.com/gcgaines/laser_lef/pull/1.

—
Reply to this email directly or view it on GitHubhttps://github.com/gcgaines/laser_lef/pull/1#issuecomment-182661933.

gcgaines commented:
I do see it, and "forked" it, but I can't seem to clone the repo to my machine. It gives me a password error in the CLI every time I try. Also, perhaps because I followed the GitHub tutorial which never referenced R Studio, I don't have the Git version control dropdowns in R Studio the way you did...

---

From: dlraffleck notifications@github.com
Sent: Thursday, February 11, 2016 1:34 PM
To: gcgaines/laser_lef
Cc: gcgaines
Subject: Re: [laser_lef] Changed some file names (#1)

Check your bitbucket account and you should see a project called IdahoPSPs...

From: gcgaines [mailto:notifications@github.com]
Sent: Wednesday, February 10, 2016 5:58 PM
To: gcgaines/laser_lef
Cc: Affleck, David
Subject: Re: [laser_lef] Changed some file names (#1)

Changes changed back.

Also, I made a BitBucket account, username = gcgaines

---

From: dlraffleck notifications@github.com
Sent: Tuesday, February 09, 2016 2:01 PM
To: gcgaines/laser_lef
Subject: [laser_lef] Changed some file names (#1)

Just changed a few file names/directories in the first R script. Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

---

You can view, comment on, or merge this pull request online at:

https://github.com/gcgaines/laser_lef/pull/1

[https://avatars2.githubusercontent.com/u/16769852?v=3&s=400]https://github.com/gcgaines/laser_lef/pull/1

Changed some file names by dlraffleck · Pull Request #1 · gcgaines/laser_lefhttps://github.com/gcgaines/laser_lef/pull/1
github.com
Just changed a few file names/directories in the first R script. Why not go ahead and accept the changes I made, then change them back and re-commit...then we'll see how this works.

Commit Summary
- Changed some file names

File Changes
- M 1.ReadRIPdb.2015dec14.Rhttps://github.com/gcgaines/laser_lef/pull/1/files#diff-0 (6)

Patch Links:
- https://github.com/gcgaines/laser_lef/pull/1.patch
- https://github.com/gcgaines/laser_lef/pull/1.diff

## 

Reply to this email directly or view it on GitHubhttps://github.com/gcgaines/laser_lef/pull/1.

## 

Reply to this email directly or view it on GitHubhttps://github.com/gcgaines/laser_lef/pull/1#issuecomment-182661933.

## 

Reply to this email directly or view it on GitHubhttps://github.com/gcgaines/laser_lef/pull/1#issuecomment-183050868.
